xcodebuild.go

```diff
@@ -234,11 +241,19 @@ func createXcodebuildTestArgs(params models.XcodeBuildTestParamsModel, xcodeMajo
 		xcodebuildArgs = append(xcodebuildArgs, "GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES", "GCC_GENERATE_TEST_COVERAGE_FILES=YES")
 	}
 
-	if xcodeMajorVersion >= 13 && params.RetryTestsOnFailure {
+	switch params.TestRepetitionMode {
```

Comment:
Xcode 13 and above only.

Reply:
You are right!

I think it makes sense to validate both configs (this and `RetryTestsOnFailure`) only being used on their respective stacks in `ProcessConfig()`, so you don't have to do it at this point.

Reply:
That sounds good, i would think about introducing a enum for example that describes which branch should we take, so conflicting options can not be selected even by accident.